lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -2089,4 +2072,31 @@ protected function switchPersisterContext($offset, $limit)
 
         $this->currentPersisterContext = $this->limitsHandlingContext;
     }
+
+    /**
+     * Retrieves class identifiers types
+     *
+     * @param ClassMetadata $class
+     * @return string[]
```

Comment:
This doesn't seem to be accurate. When I changed the array_map as such:
```php
function (string $identifier) use ($class) : string {
```
I got test failures coming from Doctrine\Tests\ORM\Functional\Ticket\DDC2579Test.

Can you please investigate? Thanks. :)

Reply:
I'll look into it!

Reply:
I updated the return types!